quickstarts/New_in_002.ipynb

```diff
@@ -0,0 +1,1351 @@
+{
```

Comment:
<p>"in teh arguments" -&gt; "in the arguments"</p><p>"python" -&gt; "Python"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/299/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Fixed.